Misc/NEWS.d/next/Library/2017-12-04-10-14-23.bpo-32173.e0C5dF.rst

```diff
@@ -0,0 +1 @@
+linecache.py add lazycache to __all__ and use dict.clear to clear the cache
```

Comment:
The news entry should be improved. The bpo issue title is not always appropriate as a News entry.

Your news says adding lazycache and use dict.clear to clear the cache, but this patch actually does a little more than those two.

Use reST markup, and end the sentence with a period.